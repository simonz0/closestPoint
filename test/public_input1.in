5
5
3.73 16.71
3.55 2.64
9.52 5.42
1.57 11.41
2.76 6.76
6
3.80 3.35
16.13 12.22
1.84 13.79
17.60 1.25
6.96 12.73
2.26 4.13
7
2.10 9.57
5.13 17.67
18.84 12.50
19.66 8.26
10.23 3.39
4.96 13.79
6.03 14.48
8
19.20 7.60
5.89 1.96
14.36 9.69
5.31 10.49
1.91 7.15
4.28 19.51
8.40 11.25
12.24 10.66
9
15.38 14.34
0.23 0.51
12.01 19.08
13.00 11.67
7.33 3.24
15.06 12.30
17.02 1.09
6.78 16.23
8.69 12.67
