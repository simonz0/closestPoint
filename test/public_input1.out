4.1951
1 4
1.7263
0 5
1.2732
5 6
2.3314
2 7
2.0649
0 5
